staging/src/slime.io/slime/modules/lazyload/charts/templates/cluster-global-sidecar.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     - name: http-{{ . }}
       port: {{ int . }}
       protocol: TCP
-      targetPort: {{ int . }}
+      targetPort: 80
```

Comment:
get it from config rather than hardcode it everywhere.

we can give it a special name.

Reply:
add param `component.globalSidecar.port`